cmake/compilers/GNU.cmake

```diff
@@ -48,6 +60,11 @@ if ("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "mips")
     set(TBB_TEST_COMPILE_FLAGS ${TBB_TEST_COMPILE_FLAGS} -DTBB_TEST_LOW_WORKLOAD)
 endif()
 
+if (MINGW)
+    set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -U __STRICT_ANSI__ -D_WIN32_WINNT=0x0601)
+    set(TBB_TEST_COMPILE_FLAGS ${TBB_TEST_COMPILE_FLAGS} -Wa,-mbig-obj)
```

Comment:
Shouldn't it be added into `Clang.cmake` as well?

Reply:
Clang+LLD on Windows defaults to outputting big object files by default, so we don't need the flag.
Speaking of object files, currently gcc fails to compile `test_eh_flow_graph` on Windows, due to the linker complaining about string table overflow. We might have to turn on optimizations for the test, to reduce the symbol length.

Reply:
`bigobj` is Windows specific. We already use it for the MSVC compiler, so I do not see any issues to use it for gcc as well.